service/history/queue/transfer_queue_processor.go

```diff
@@ -601,6 +601,7 @@ func newTransferQueueFailoverProcessor(
 
 	currentClusterName := shardContext.GetService().GetClusterMetadata().GetCurrentClusterName()
 	failoverUUID := uuid.New()
+	// TODO: unused, find log and restore or remove this With
```

Comment:
if unused, can consider removing code?

Reply:
yea, I honestly just forgot that I added this as I was reading through.
lemme see what the history of this logger is, I wonder if it used to log something that would still be useful sometimes.

Reply:
nope, nvm, it's just wrong.  the logger is being passed to the `newTransferQueueProcessorBase` at the end.